controllers/util.go

```diff
@@ -8,6 +8,7 @@ import (
 	"github.com/Nordix/eno/pkg/l2serviceattachmentparser"
 	"github.com/Nordix/eno/pkg/render"
 	"github.com/go-logr/logr"
+	"github.com/prometheus/common/log"
```

Comment:
Why do we need this ?

Reply:
yes i removed it. i think it got added unintentionally.